packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -1441,4 +1441,25 @@ describe('Dropdown', () => {
       expect(stopPropagation).toBeCalledTimes(1)
     })
   })
+
+  describe('renderSelectedItem', () => {
+    it('calls renderSelectedItem in multiple selection', () => {
+      const renderSelectedItem = jest.fn((SelectedItem, props) => (
+        <SelectedItem {...props} header={`${props.header} (selected)`} />
+      ))
```

Comment:
There are two options:
- simplify `jest.fn`:  `jest.fn(() => null)` as we don't test rendered content
- expand this UT to catch that these elements were rendered:
```tsx
      const className = faker.random.word()
      const renderSelectedItem = jest.fn((SelectedItem, props) => (
        <SelectedItem {...props} className={className} header={`${props.header} (selected)`} />
      ))
//

expect(wrapper.find(`.${className}`)).toHaveLength(5)
```